windows/installer.nsi

```diff
@@ -212,6 +218,30 @@ Section "Install_64Bit" INST_64
                 MessageBox MB_OKCANCEL|MB_ICONEXCLAMATION "$(OLD_VER_REMOVE_FAILED)" /sd IDCANCEL IDOK continueinstall
                 goto quit
             ${EndIf}
+
+            ; ask to move old ASIO4ALL registry configuration
+
+            ; check if old ASIO4ALL keys exit
+
+            ClearErrors
+            EnumRegKey $0 HKCU "SOFTWARE\ASIO4ALL v2 by Wuschel\8A9E7A56" 0
```

Comment:
Check if the key of the old version of A4A exists for the currently logged in user (not really a good way to check that. The currently logged in user might not have started Jamulus + A4A yet)